examples/gradle-kotlin-dsl/src/main/kotlin/Test.kt

```diff
@@ -0,0 +1,13 @@
+package incorrect
+
+class incorrectname: Exception {
```

Comment:
if possible - let's copy our smoke test here?

Reply:
We probably need to create a good test source for examples: so that users can understand diktat capabilities. Let's create a separate issue for this.